src/main/build/debug.c

```diff
@@ -98,4 +98,5 @@ const char * const debugModeNames[DEBUG_COUNT] = {
     "RX_TIMING",
     "D_LPF",
     "VTX_TRAMP",
+    "GHST"
```

Comment:
Please add the trailing comma. This minimizes the `diff` when future enhancements are made to this array.